README.md

```diff
@@ -6,25 +6,47 @@
 
 1. Add the following to your **root** `build.gradle` file:
 
-```
-allprojects {
-    repositories {
-        ...
+    ```
+    allprojects {
+        repositories {
+            ...
 
-        maven { url "https://raw.githubusercontent.com/placed/android-placed-sdk/master/repository" }
+            maven { url "https://raw.githubusercontent.com/placed/android-placed-sdk/master/repository" }
+        }
     }
-}
-```
+    ```
 
 2. Add the following to your **app** `build.gradle` file:
 
-```
-dependencies {
-    ...
+    ```
+    dependencies {
+        ...
 
-    compile 'com.placed.client:android-persistent-sdk:1.30'
-}
-```
+        compile 'com.placed.client:android-persistent-sdk:1.30'
```

Comment:
Given that our API itself is stable and not likely to change, I like your idea. The Android Studio linter, however, complains when you use this:
>Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds.

Do you have any concerns with that showing up in affiliate's builds?